### ISSUE TITLE
Autodetection broken ?

### ISSUE THREAD

**Issue by adepolas** [alchemy-fr/BinaryDriver#11]
I had the [error](https://github.com/PHP-FFMpeg/PHP-FFMpeg/issues/352#issuecomment-294120455) "Unable to load FFMpeg" while using the [PHP-FFMPeg](https://github.com/PHP-FFMpeg/PHP-FFMpeg) driver.

Looks like the auto-detection failed,
https://github.com/alchemy-fr/BinaryDriver/blob/master/src/Alchemy/BinaryDriver/AbstractBinary.php#L160-L162

Is the package working only under Windows ? Got any ideas ?
(As I've tried in both Linux/Mac environment)

**Comment by jens1o** [comment]
No, it's not only working on Windows. In the tests, it's working too with the automatic detection(and they run under linux).
